frontend/src/hooks/useGamePlay.ts

```diff
@@ -32,8 +34,8 @@ export default function useGamePlay(
   const socket = useContext(SocketContext);
   const queryClient = useQueryClient();
 
-  const interval = useRef<NodeJS.Timer | null>(null);
-  const animationFrame = useRef<number | null>(null);
+  const countInterval = useRef<NodeJS.Timer | null>(null);
+  const drawInterval = useRef<NodeJS.Timer | null>(null);
```

Comment:
```suggestion
  const countInterval = useRef<ReturnType<typeof setInterval> | null>(null);
  const drawInterval = useRef<ReturnType<typeof setInterval> | null>(null);
```

https://stackoverflow.com/questions/51376589/typescript-what-type-is-setinterval